package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.1.2",
+  "version": "5.1.4",
```

Comment:
You should only increase by 1.

I know: your other PR might get merged first. But if it does then you'll need to rebase anyway, so this isn't saving you any work. (Sorry.)

Reply:
reverted back to 5.1.3